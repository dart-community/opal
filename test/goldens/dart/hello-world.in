const String message = 'Hello, world!';

void main() {
  // Print the message.
  print('The message is: $message');
}
