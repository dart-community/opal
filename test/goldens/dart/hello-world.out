const String message = 'Hello, world!';
^^^^^
source-code-dart-keyword-const
     ^
     source-code-dart-whitespace
      ^^^^^^
      source-code-dart-identifier-type-built-in
            ^
            source-code-dart-whitespace
             ^^^^^^^
             source-code-dart-identifier
                    ^
                    source-code-dart-whitespace
                     ^
                     source-code-dart-keyword-operator
                      ^
                      source-code-dart-whitespace
                       ^
                       source-code-dart-literal-string-literal-string-begin
                        ^^^^^^^^^^^^^
                        source-code-dart-literal-string-literal-string-unquoted
                                     ^
                                     source-code-dart-literal-string-literal-string-end
                                      ^
                                      source-code-dart-punctuation

void main() {
^^^^
source-code-dart-identifier-type-built-in
    ^
    source-code-dart-whitespace
     ^^^^
     source-code-dart-identifier-function
         ^
         source-code-dart-punctuation
          ^
          source-code-dart-punctuation
           ^
           source-code-dart-whitespace
            ^
            source-code-dart-punctuation
  // Print the message.
^^
source-code-dart-whitespace
  ^^^^^^^^^^^^^^^^^^^^^
  source-code-dart-comment-line
  print('The message is: $message');
^^
source-code-dart-whitespace
  ^^^^^
  source-code-dart-identifier-function
       ^
       source-code-dart-punctuation
        ^
        source-code-dart-literal-string-literal-string-begin
         ^^^^^^^^^^^^^^^^
         source-code-dart-literal-string-literal-string-unquoted
                         ^^^^^^^^
                         source-code-dart-literal-string-literal-string-unquoted-literal-string-interpolation
                                 ^
                                 source-code-dart-literal-string-literal-string-end
                                  ^
                                  source-code-dart-punctuation
                                   ^
                                   source-code-dart-punctuation
}
^
source-code-dart-punctuation
